build.gradle

```diff
@@ -123,6 +127,19 @@ subprojects { subproject ->
 		testCompile "org.springframework:spring-test:$springVersion"
 		testCompile "org.slf4j:slf4j-log4j12:$slf4jVersion" // amqp-client now uses SLF4J
 		testRuntime "org.apache.logging.log4j:log4j-jcl:$log4jVersion"
+
+		testCompile "org.junit.jupiter:junit-jupiter-api:${junitJupiterVersion}"
+		testRuntime "org.junit.jupiter:junit-jupiter-engine:${junitJupiterVersion}"
+		testRuntime "org.junit.platform:junit-platform-commons:${junitPlatformVersion}"
+		testRuntime "org.junit.platform:junit-platform-launcher:${junitPlatformVersion}"
+
+		// To support JUnit 4 tests
+		testCompile "junit:junit:${junit4Version}"
```

Comment:
We have this one already in the beginning of this `dependencies` block with the exclude for the `hamcrest`